create-release/action.yml

```diff
@@ -59,11 +62,21 @@ runs:
       shell: bash
       run: |
         git config --global user.name "GitHubActions"
+        git config --global user.email "github-actions@users.noreply.github.com"
         npm config set git-tag-version false
-    - name: Update main package.json
-      if: inputs.update-root-package_json == 'true'
+    - name: Update main package.json via npm
+      if: inputs.update-root-package_json == 'true' && inputs.projen != 'true'
       shell: bash
       run: npm version ${{ steps.bump-version.outputs.new_tag }}
+    - name: Update main package.json via projen
+      if: inputs.update-root-package_json == 'true' && inputs.projen == 'true'
+      shell: bash
+      run: |
+        fileName=$([ -f .projenrc.ts ] && echo ".projenrc.ts" || echo ".projenrc.js")
+        regex="package.addField('version', '[0-9]\.[0-9]\.[0-9]')"
+        newVersion=$(echo ${{ steps.bump-version.outputs.new_tag }} | cut -c 2-6)
+        sed -i -e "s/$regex/package.addField('version', '$newVersion')/" $fileName
```

Comment:
@quesabe just thought — can we maybe do it like this in `.projenrc.ts`:
```typescript
project.addField('version', process.env.PACKAGE_VERSION || <read from package.json>)
```

What do you think about such approach?

Reply:
Sounds promising - that gonna simplify the action code a lot. Let me try it and see if there is any downside to this.